### PR TITLE
Add talk about Logux

### PR DESCRIPTION
**Pull request by ai**
Hi. I had this talk of React London, but didn’t see too many person from Berlin there.

Slides: http://slides.com/ai/logux

### CODE REVIEW

File: speakers.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [
   {
-    "name": "You?",
-    "uri": "https:\/\/twitter.com\/berlinjs",
-    "title": "Your talk title",
-    "description": "This is your spot to take! We want to hear your story. Get in touch with us and we're happy to host you!"
+    "name": "Andrey Sitnik",
+    "uri": "https://twitter.com/andreysitnik",
```
Comment (kof) [inline speakers.json]: not sure why they escape in plain text, but you should probably keep the backslashes)

File: speakers.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+    "title": "Logux, a new way of client-server conversation Andrey Sitnik",
```
Comment (kof) [inline speakers.json]: Logux - a new way of client-server conversation, by Andrey Sitnik.

**Comment by rmehner** [comment]
@ai hey there, I've read something about you rebooking from Copenhagen. Are you still able to make it? If, so, we'd like to have you on stage :)

**Comment by ai** [comment]
Awesome! I will rebook my flight tickets today 👍

**Comment by ai** [comment]
@rmehner done, I will be in Berlin at May 4 :)

**Pull request closed without merging**